file-share-service-desktop-client/Modules/Search/SearchViewModel.cs

```diff
@@ -89,12 +96,21 @@ private async Task ExecuteSearch()
             batchDetailsVM = new List<BatchDetailsViewModel>();
             try
             {
+                logger.LogInformation("File Share Service search started for SearchText :{searchText}", searchText);
                 var fssClient = fileShareApiAdminClientFactory.Build();
-                var result = await fssClient.Search(searchText, pageSize, pageOffset);
+                var result = await fssClient.Search(searchText, pageSize, pageOffset, CancellationToken.None);
 
-                SearchResultAsJson = result.ToJson();
-                SearchResult = result;
-                foreach(var entries in  SearchResult.Entries)
+                if (!result.IsSuccess)
```

Comment:
Please check for the status code in case of failure.
if status code is 400
display & log same error what you receive from Batch Search API endpoint.
else
display some common message like "Something went wrong, please try again later or contact administrator."
log same error what you receive from Batch Search API endpoint.

Reply:
resolved